src/test-utils/constants.tsx

```diff
@@ -0,0 +1,29 @@
+import { CurrencyAmount, Token } from '@uniswap/sdk-core'
+import { FeeAmount, Pool } from '@uniswap/v3-sdk'
+import JSBI from 'jsbi'
+
+export const testToken1 = new Token(1, '0x0000000000000000000000000000000000000001', 18)
```

Comment:
nit: I prefer constants to be in caps (eg all of the test tokens and pools):

```suggestion
export const TEST_TOKEN_1 = new Token(1, '0x0000000000000000000000000000000000000001', 18)
```